### PR TITLE
Add approve, revoke subcommands

### PR DESCRIPTION
**Pull request by CriesofCarrots**
Toward #502

### CODE REVIEW

File: token/cli/src/main.rs [hunk 1]
```diff
@@ -482,6 +483,94 @@ fn command_unwrap(config: &Config, address: Pubkey) -> CommandResult {
...
+        .rpc_client
+        .get_token_account_balance_with_commitment(&account, config.commitment_config)?
+        .value;
+    let source_account = config
+        .rpc_client
+        .get_account_with_commitment(&account, config.commitment_config)?
+        .value
+        .unwrap_or_default();
+    let data = source_account.data.to_vec();
+    let mint_pubkey = Account::unpack_from_slice(&data)?.mint;
```
Comment (t-nelson) [inline token/cli/src/main.rs:507]: This can be replaced by one call to `get_token_account_with_commitment()`

File: token/cli/src/main.rs [hunk 2]
```diff
@@ -482,6 +483,94 @@ fn command_unwrap(config: &Config, address: Pubkey) -> CommandResult {
...
+}
+
+fn command_revoke(config: &Config, account: Pubkey) -> CommandResult {
+    let source_account = config
+        .rpc_client
+        .get_account_with_commitment(&account, config.commitment_config)?
+        .value
+        .unwrap_or_default();
+    let data = source_account.data.to_vec();
+    let delegate = Account::unpack_from_slice(&data)?.delegate;
```
Comment (t-nelson) [inline token/cli/src/main.rs:542]: `get_token_account_with_commitment()` can spare you the data wrangling/unpacking here too

**Review by t-nelson — APPROVED** [review]
LGTM!  Just some simplifications should you choose

File: token/cli/src/main.rs (hunk 1 above)
Comment (CriesofCarrots) [reply token/cli/src/main.rs:507]: Oh, totally! Thanks for the reminder. I'll fixup `transfer` while I'm on that job.

**Review by t-nelson — APPROVED** [review]

**Pull request merged**